pkg/controller/nodelifecycle/scheduler/taint_manager.go

```diff
@@ -115,6 +119,27 @@ func deletePodHandler(c clientset.Interface, emitEventFunc func(types.Namespaced
 	}
 }
 
+func addConditionAndDeletePod(ctx context.Context, c clientset.Interface, name, ns string) (err error) {
+	if feature.DefaultFeatureGate.Enabled(features.JobPodFailurePolicy) {
+		pod, err := c.CoreV1().Pods(ns).Get(ctx, name, metav1.GetOptions{})
+		if err != nil {
+			return err
+		}
+		newStatus := pod.Status.DeepCopy()
+		if apipod.UpdatePodCondition(newStatus, &v1.PodCondition{
+			Type:    "TargettedForDisruption",
```

Comment:
even if we are not exporting the condition, you can still have it as a constant on each of the packages that need it, so you can reuse it for tests.

And it's `Targeted`

Reply:
updated